packages/toolpad-core/src/Account/SignInButton.tsx

```diff
@@ -25,6 +25,8 @@ export /**
       sx={{
         textTransform: 'capitalize',
         filter: 'opacity(0.9)',
+        width: '50%',
+        margin: (theme) => `${theme.spacing(1)} auto`,
```

Comment:
I might have just added a margin but this looks ok too.
Also maybe `mh: 1` to avoid using a function + interpolating the string.